src/components/VocabularyListItem.tsx

```diff
@@ -2,11 +2,11 @@ import React, { ReactElement } from 'react'
 import { widthPercentageToDP as wp } from 'react-native-responsive-screen'
```

Comment:
This is why I don't really like route specific component folders, once it is used in multiple routes it has to be moved/renamed.

Reply:
yeah there are different opinions about this which is definitely fine. I personally find it worse that I have to look in multiple folders to find the right file than to find the file in a large folder.

Reply:
> 

it is now merged. you can base on it